python/requirements/all.txt

```diff
@@ -0,0 +1,16 @@
+numpy
```

Comment:
I'd recommend adding comments at the top of each of these files to indicate what they are used for, since it may not always be obvious from the filename. PIP requirements files support `#` comments.https://pip.pypa.io/en/latest/reference/pip_install/#requirements-file-format

Reply:
I added a way to specify a comment to each piece and that's now placed at the top of each requirements.txt file